lib/cadet_web/router.ex

```diff
@@ -112,18 +129,25 @@ defmodule CadetWeb.Router do
     )
 
     get("/users", AdminUserController, :index)
-    post("/users/:userid/goals/:uuid/progress", AdminGoalsController, :update_progress)
+    put("/users", AdminUserController, :upsert_users_and_groups)
+    put("/users/role", AdminUserController, :update_role)
+    delete("/users", AdminUserController, :delete_user)
 
     put("/achievements", AdminAchievementsController, :bulk_update)
+    # update may be unused
     put("/achievements/:uuid", AdminAchievementsController, :update)
     delete("/achievements/:uuid", AdminAchievementsController, :delete)
 
+    post("goals/:uuid/progress/:course_reg_id", AdminGoalsController, :update_progress)
     get("/goals", AdminGoalsController, :index)
     put("/goals", AdminGoalsController, :bulk_update)
     put("/goals/:uuid", AdminGoalsController, :update)
     delete("/goals/:uuid", AdminGoalsController, :delete)
 
-    put("/settings/sublanguage", AdminSettingsController, :update)
+    put("/config", AdminCoursesController, :update_course_config)
+    get("/config/assessment_configs", AdminCoursesController, :get_assessment_configs)
+    put("/config/assessment_configs", AdminCoursesController, :update_assessment_configs)
+    delete("/config/assessment_config", AdminCoursesController, :delete_assessment_config)
```

Comment:
This should probably be renamed to `assessment_types`

Also the delete should have the assessment type in the URL: `/config/assessment_types/:type_name`

Reply:
I could put the id in the url since name is not unique. However, should I keep this name so as to be consistent with the other routes related to assessment config?